src/Client.php

```diff
@@ -141,6 +141,16 @@ public function create(AutotaskObjects\Entity $obj)
         $params = new AutotaskObjects\CreateParam($obj);
         return $this->_call('create', array($params));
     }
+    
+    public function bulkCreate(array $objs)
+    {
+        $createObjs;
+        foreach ($objs as $obj) {
+           if (!$createObjs) $createObjs = new AutotaskObjects\CreateParam($obj);
```

Comment:
All if statements, even if just 1 line, should be wraped in `{ }`, and broken out into multiple lines.
```
if (!$createObjs) {
    $createObjs = new AutotaskObjects\CreateParam($obj)
} else {
    $createObjs->Entites[] = $obj
}